ui/lib/apps/SearchLogs/utils/index.ts

```diff
@@ -1,142 +1,34 @@
-import {
-  ClusterinfoClusterInfo,
-  LogsearchTaskGroupResponse,
-  LogsearchTaskModel,
-  ModelRequestTargetNode,
-} from '@lib/client'
-import { TimeRange } from '@lib/components'
-
-export const DATE_TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss'
+export enum LogLevel {
+  Unknown = 0,
+  Debug,
+  Info,
+  Warn,
+  Trace,
+  Critical,
+  Error,
+}
 
-export const LogLevelMap = {
-  0: 'UNKNOWN',
-  1: 'DEBUG',
-  2: 'INFO',
-  3: 'WARN',
-  4: 'TRACE',
-  5: 'CRITICAL',
-  6: 'ERROR',
+export const LogLevelText = {
+  [LogLevel.Unknown]: 'UNKNOWN',
+  [LogLevel.Debug]: 'DEBUG',
+  [LogLevel.Info]: 'INFO',
+  [LogLevel.Warn]: 'WARN',
+  [LogLevel.Trace]: 'TRACE',
+  [LogLevel.Critical]: 'CRITICAL',
+  [LogLevel.Error]: 'ERROR',
 }
```

Comment:
It seems ok to define the LogLevelText as an Array, likes:

```
export const LogLevelText = ['UNKNOWN', 'DEBUG' ...]
```

But this is not a big deal, just simplify the code slightly.

Reply:
It relies on the value of log levels to work, might not be very ideal.